core-primitives/types/src/lib.rs

```diff
@@ -36,6 +34,9 @@ pub type PalletString = Vec<u8>;
 #[cfg(feature = "std")]
 pub type PalletString = String;
 
+pub use sp_core::{crypto::AccountId32 as AccountId, H256};
+pub use substrate_api_client::{AccountData, AccountInfo};
```

Comment:
Re-export useful types from the api-client, so we only need to import our types crate if we need them.

Reply:
And these substrate types are only in the API client? Just from the naming, it seems a bit heavy to me to import an API client for re-exporting only primitive types? But if those types are only defined in the API client, then so be it.

Reply:
Yes, we redefined them in the client because they are private in substrate...

Reply:
we could however, redefine them here. Might even make sense.

Reply:
Yep.

Reply:
ah I see - then maybe it would make sense in the future to have these types in more light-weight primitives/types kind of crate?

Reply:
You are right. There is an ancient issue in the api-client demanding to split it in several crates. 👍